specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/mfe.json

```diff
@@ -11866,6 +11866,59 @@
           "final-state-via": "location"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{workspaceName}/serverlessEndpoints/{name}/status": {
+      "post": {
+        "tags": [
+          "ServerlessEndpoint"
+        ],
+        "summary": "Temporary status API until integration completes.",
```

Comment:
>Temporary status API until integration completes. [](http://example.com/codeflow?start=20&length=49)

This doesn't sound like an API that's intended to ship... what's the plan?

Reply:
We do need this API to ship. We will update the summary.